### PR TITLE
DOCS-1994 Bubble charts

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 1]
```diff
@@ -45,8 +45,34 @@ If denylisted or graylisted IP actually does not belong to a malicious bot, you
...
+* **Bot scope** displays bot relation to its targets, including:
+
+    * Bubble size: the more different API endpoints the IP requested, the larger the size.
+    * Color: the more requests with unsafe method the IP requested, the closer to the red.
+    * Horizontally: the more RPS (requests per second), the farther to the right.
+    * Vertically: the more error responses (4XX, 5XX) from the origin server, the higher on the graph.
+
 The **API Abuse Prevention** module compiles client traffic into URL patterns. The URL pattern may have the following segments:
 
 | Segment | Contains | Example |
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:78]: the screenshot displays the buttin "MArk as good bot" - it is not a correct button name

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 2]
```diff
@@ -45,18 +45,46 @@ If denylisted or graylisted IP actually does not belong to a malicious bot, you
...
-| FILE | Non-interactive files. | `image.png` |
+| STATIC | The folders that contain static files of different kinds. | `images`, `js`, `css` |
+| FILE | Static file names. | `image.png` |
 | QUERY | Query parameters. | - |
 | AUTH | Content related to the authentication/authorization endpoints. | - |
-| LANGUAGE | Language-related segments. | `en`, `fr` |
+| LANGUAGE | Language-related parts. | `en`, `fr` |
 | HEALTHCHECK | Content related to the health check endpoints. | - |
-| VARY | Other data not important for analysis. | - |
+| VARY | The segment is marked as VARY if it is impossible to attribute it to other categories. A variable part of the URL path. | - |
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:90]: did you change the [denylisted-bot-ips.png](https://github.com/wallarm/product-docs-en/pull/791/files#diff-f8f526aac7a68d1b6b91c9cd8076dffc0a73bad8504c894380afef8164025972) image? if not, then why is it displayed in diff?

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 3]
```diff
@@ -45,18 +45,46 @@ If denylisted or graylisted IP actually does not belong to a malicious bot, you
...
+On the plots, you can also compare you current bot (**this bot**) with the other bots that attacked the same application within the past 24 hours. If too many bots did that, only 30 most suspicious will be displayed.
+
+The bubble plots:
+
+* **Bot performance** displays intensity of bot activities, including:
+
+    * Bubble size: request non-uniqueness, the more the IP requested the same (not unique) API endpoints, the larger the size.
+    * Color: scheduled requests, the more the IP requested API endpoints due to the schedule (the same time intervals), the closer to the red.
+    * Horizontally: the more RPS (requests per second), the farther to the right.
+    * Vertically: the higher the request rate (how fast the IP sends the requests), the higher on the graph. Comparing to RPS: IP can send 3 requests per second (which is not a lot), but do that within 3 milliseconds (which is very fast).
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:62]: "the higher the request rate (how fast the IP sends the requests), the higher on the graph." is incorrect form. subject is missed in the 2nd part. please check all these coolocations

File: docs/latest/user-guides/api-abuse-prevention.md [hunk 4]
```diff
@@ -45,18 +45,46 @@ If denylisted or graylisted IP actually does not belong to a malicious bot, you
...
 | IDENTIFIER | Various identifiers like numeric identifiers, UUIDs, etc. | - |
-| STATIC | Static data. | images, js |
-| FILE | Non-interactive files. | `image.png` |
+| STATIC | The folders that contain static files of different kinds. | `images`, `js`, `css` |
+| FILE | Static file names. | `image.png` |
 | QUERY | Query parameters. | - |
 | AUTH | Content related to the authentication/authorization endpoints. | - |
-| LANGUAGE | Language-related segments. | `en`, `fr` |
+| LANGUAGE | Language-related parts. | `en`, `fr` |
 | HEALTHCHECK | Content related to the health check endpoints. | - |
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-abuse-prevention.md:89]: did you take the [api-abuse-events.png](https://github.com/wallarm/product-docs-en/pull/791/files#diff-6584c7e4f8ab509964e8f1bace0870f81e049f3182de7586e6182450cbf793d0) screenshot from local build? why does it display "Active threat verification" in 3 lines? it looks bad

Comment (AnastasiaTWW) [reply docs/latest/user-guides/api-abuse-prevention.md:89]: it also shows jira domains, it is sensitive data

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 1 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-abuse-prevention.md:80]: The button is not presented now. Can we close the comment?

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 2 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-abuse-prevention.md:90]: It was changed twice: A > B > A :-)

File: docs/latest/user-guides/api-abuse-prevention.md (hunk 4 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-abuse-prevention.md:89]: See details = 
https://docs.wallarm.com/user-guides/api-abuse-prevention/#exploring-blocked-malicious-bots-and-their-attacks

**Pull request merged**